samples/DataGridListSample.js

```diff
@@ -16,7 +16,7 @@ enyo.kind({
 			{kind: "moon.ToggleButton", content:"Selection", name:"selectionToggle"},
 			{kind: "moon.ContextualPopupDecorator", components: [
 				{kind: "moon.ContextualPopupButton", content:"Selection Type"},
-				{kind: "moon.ContextualPopup", classes:"moon-4h moon-6v", components: [
```

Comment:
Yeah it didn't seem necessary as there's no scroller in the popup - I'm open to leaving it in though.